### PR TITLE
Homework 6 1 heghine khachatryan

### PR DESCRIPTION
**Pull request by HeghineKhachatryan**
changed types of classes into enum

### CODE REVIEW

File: src/pizzeria/TestOrders.java [hunk 1]
```diff
@@ -4,11 +4,12 @@
     public static void main(String[] args) {
         CheckPrinter checkPrinter = new CheckPrinter();
         Customer customer = new Customer(7717, "Heghine","+(374 94)374374");
-        Ingredient ingredientsToAdd = new Ingredient("Pepper", 0.4);
-        Ingredient[] ingredients = {ingredientsToAdd, Ingredient.getCheese(), Ingredient.getCorn(), Ingredient.getTomatoPaste()};
-        PizzaType pizzaType = new PizzaType("Calzone");
+        Ingredient[] ingredients = {Ingredient.CHEESE, Ingredient.CORN, Ingredient.TOMATO_PASTE};
+        PizzaType pizzaType = PizzaType.CLOSED;
```
Comment (GevKishmishyan) [inline src/pizzeria/TestOrders.java:8]: No need to create a variable here. You can directly use enum inside **new Pizza();**

File: src/pizzeria/Pizza.java [hunk 2]
```diff
@@ -22,32 +22,37 @@ public Pizza(String name, PizzaType pizzaType) {
...
-
         for (Ingredient ingredient : ingredients) {
             price += ingredient.getPrice();
         }
-
         return price;
     }
 
-    public void addIngredients(Ingredient ingredient) {
+    public void addIngredients (Ingredient ingredients) {
```
Comment (GevKishmishyan) [inline src/pizzeria/Pizza.java:47]: Change this method name, because you add **ONE** ingredient -> **addIngerdient(Ingredient ingredient)**

**Review by GevKishmishyan — COMMENTED** [review]
Check these small comments, the others are OK.

File: src/pizzeria/Pizza.java (hunk 2 above)
Comment (HeghineKhachatryan) [reply src/pizzeria/Pizza.java:47]: done

File: src/pizzeria/TestOrders.java (hunk 1 above)
Comment (HeghineKhachatryan) [reply src/pizzeria/TestOrders.java:8]: done

**Pull request closed without merging**